middleware/packet-forward-middleware/router/keeper/keeper.go

```diff
@@ -202,7 +202,8 @@ func (k *Keeper) ForwardTransferPacket(
 	inFlightPacket *types.InFlightPacket,
 	srcPacket channeltypes.Packet,
 	srcPacketSender string,
-	receiver string,
+	feePayer string,
```

Comment:
Why is this called fee payer? AFAIK the fee is paid by the relayer everywhere except for the initial send

Reply:
oh, I see! this is in the case where there is a forwarding fee. Interesting 🤔 I wonder if any chain has the fees enabled.